src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_access_restriction_commands.py

```diff
@@ -81,7 +81,7 @@ def test_functionapp_access_restriction_set_complex(self, resource_group, locati
             JMESPathCheck('scmIpSecurityRestrictionsUseMain', False)
         ])
 
-    @ResourceGroupPreparer(parameter_name_for_location='location', location=WINDOWS_ASP_LOCATION_WEBAPP)
+    @ResourceGroupPreparer(random_name_length=17, parameter_name_for_location='location', location=WINDOWS_ASP_LOCATION_WEBAPP)
```

Comment:
Please add a comment here this is temporary until the bug fix in the API is deployed successfully & should be removed.

Reply:
Why random_name_length=17 can fix it?

Reply:
This is because we had a regression in our API with the length of the RG - @ThejaChoudary I believe the fix for this is already deployed -so we can remove this hack on the test and re-try.